ait/core/server/plugins/data_archive.py

```diff
@@ -75,7 +75,7 @@ def process(self, input_data, topic=None, **kwargs):
             **kwargs:    any args required for connected to the backend
         """
         try:
-            load = pickle.loads(eval(input_data))
+            load = pickle.loads(input_data)
```

Comment:
@MJJoyce I haven't had a deep look, but might input_data already have been de-pickled by the server_util.decode()?  Or are the contents of the message pickled twice before being pushed onto 0Mq?

Reply:
I realized I never copy and pasted the answer to this here from our offline chat:

```
currently our packet handlers are pickling the contents before they send them off
so we have (packet id, pickled content) which gets sent to the “client sender”, which pickles it again
```

I'll make sure this gets handled in a follow up ticket